main.py

```diff
@@ -1,116 +1,99 @@
-# Import Pyton Libraries ---------------------------------------------------
-import os
+#!/usr/bin/env python3
+from os import path, getenv, listdir
 import subprocess
+import logging
+import argparse
 
-# Logging ------------------------------------------------------------------
-q0 = input('''
```

Comment:
Executed what function?? I ran a program.

"In this directory..."? In what directory this current directory? They only acceptable answers are Yes or yes? So I have no choice wether or not to proceed? Why even ask me? 

Let's remove that.